river/misc/cms.py

```diff
@@ -119,7 +125,7 @@ class CountMin:
     And use it to calculate the cosine distance between the elements monitored in `cms_a` and `cms_b`
 
     >>> cosine_dist(cms_a, cms_b)
-    0.8785023776056446
+    0.17536377918173912
```

Comment:
```suggestion
    0.175363...
```

It leaves more room for numerical errors with different Python/dependency versions

Reply:
I didn't know this trick!!! Nice :D